resources/dicts/events/ceremonies/ceremony-master.json

```diff
@@ -2789,6 +2789,375 @@
+  ],
+  "warrior_68": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "loving",
+      "compassionate",
+      "calm",
+      "careful",
+      "thoughtful",
+      "sincere"
+    ],
+    "As p1 and p2 fuss, erasing each others' efforts at licking (old_name)'s fur flat, (old_name) purrs. Surrounded by {PRONOUN/m_c/poss} parents, l_n names {PRONOUN/m_c/object} m_c for the r_h {PRONOUN/m_c/subject} {VERB/m_c/have/has} shown in {PRONOUN/m_c/poss} training, welcoming {PRONOUN/m_c/object} to take {PRONOUN/m_c/poss} place as a full warrior of c_n."
+  ],
+  "warrior_69": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "charismatic",
+      "ambitious",
+      "confident",
+      "childish",
+      "righteous",
+      "arrogant",
+      "competitive",
+      "flamboyant"
+    ],
+    "(old_name) practically glows with delight, basking in the glory of {PRONOUN/m_c/poss} naming ceremony, in the attention and praise of {PRONOUN/m_c/poss} parents. l_n names {PRONOUN/m_c/object} m_c for {PRONOUN/m_c/poss} r_h as the new warrior preens with accomplishment."
+  ],
+  "warrior_70": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "charismatic",
+      "ambitious",
+      "confident",
+      "childish",
+      "righteous",
+      "arrogant",
+      "competitive",
+      "flamboyant"
+    ],
+    "(old_name) has thrown {PRONOUN/m_c/self} at every challenge in {PRONOUN/m_c/poss} path, and l_n notes {PRONOUN/m_c/subject} {VERB/m_c/have/has} the r_h of a tiger, may it serve {PRONOUN/m_c/object} well. It's time for (old_name) to take both {PRONOUN/m_c/poss} place as an adult of c_n, and {PRONOUN/m_c/poss} full name of m_c."
+  ],
+  "warrior_71": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "oblivious",
+      "calm",
+      "wise"
+    ],
+    "As the Clan yowls m_c's new warrior name {PRONOUN/m_c/subject} {VERB/m_c/smile/smiles} and {VERB/m_c/nod/nods}, pleased but not overcome. l_n jumps down to congratulate {PRONOUN/m_c/object}. As their heads press together, l_n whispers that balanced sense of self and r_h will serve m_c well."
+  ],
+  "warrior_72": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "strict",
+      "responsible"
+    ],
+    "With c_n gathered for celebration, l_n highlights (old_name)'s r_h, {PRONOUN/m_c/poss} dedication to truth and the warrior code. It is for this that l_n names {PRONOUN/m_c/object} m_c, welcoming {PRONOUN/m_c/object} as a full warrior of c_n."
```

Comment:
warrior code should be Warrior Code. also a missing oxford comma in the first sentence (r_h, their dedication , **and** the ...